docs/source/multi-dd.rst

```diff
@@ -175,6 +176,15 @@ explicit conversion mechanisms.
 .. [#ignore_type_change] These type changes are not supported. Quantities in the
     destination IDS will remain empty.
 
+.. [#ps3to4] In Data Dictionary 3.39.0 and older, all dynamic quantities in the
+    ``pulse_schedule`` IDS had their own time array. In DD 4.0.0 this was
```

Comment:
what about 3.40 etc? was it not restructured at that point?

Reply:
Correct, but IO requested to make the conversion logic available for DD3.x to 4.x.

> what about 3.39 -> 3.41? there is a bit of a gap in the conversion logic, right?

3.39 -> 3.41 follows regular conversion logic and will not resample dynamic quantities. It would be trivial to update the logic to execute when converting from DD <=3.39.0 to DD >= 3.40.0, though. @olivhoenen any thoughts?

> 3.41 -> 4?

Nothing needs to be done w.r.t. the time bases between these DD versions.

Reply:
Clear! The conversion now also applies when going to a DD version >= 3.40.0.